doc/designs/passkeys.md

```diff
@@ -0,0 +1,246 @@
+# Passkey authentication
+
+## Overview
+
+Traditional authentication with a password is not considered secure enough
+by many companies or government agencies. Alternate and more secure
+solutions exist, among which the use of passkeys, where the private
+key is stored on the device and the server only needs to know the public
+key.
+
+The goal of this feature is to use a passkey to authenticate a user
```

Comment:
Please add at this point an explanation what do we understand as a 'passkey'. In SSSD design document we have added the following statement: https://github.com/SSSD/sssd.io/pull/57/files#diff-e9f09192772572ab2c78b7cb7083c40b477533b512b583cb89deeab9aada42fdR20

> For the purpose of this feature, passkey is a FIDO2 compatible device supported by the libfido2 library.

We can link to FIDO2 landing page, https://fidoalliance.org/fido2/, to give more details.